Tests/MySQLNIOTests/MySQLNIOTests.swift

```diff
@@ -636,6 +636,19 @@ final class MySQLNIOTests: XCTestCase {
         XCTAssertEqual(time.date, nil)
     }
     
+    func testTextMySQLTimeParse() throws {
+        let conn = try MySQLConnection.test(on: self.eventLoop).wait()
+        defer { try! conn.close().wait() }
+        
+        // MariaDB 10.5 returns timestamp columns in text format.
```

Comment:
More precisely, `simpleQuery()` uses the old text resultset protocol, which always returns textual results; it's not just MariaDB.
```suggestion
        // The text protocol returns timestamp columns in text format.
```